src/extension.ts

```diff
@@ -30,36 +31,33 @@ export function activate(context: vscode.ExtensionContext) {
 	// TODO: pass this into any command function that needs to interact with azure
 	const az: AzApi = new Az(azureAccount);
 
+	const git:GitAPI = <GitAPI>vscode.extensions.getExtension('vscode.git')!.exports.getAPI(1);
```

Comment:
💡Curious: Why at the extension level? either we can error handle here and throw early or instead of parameter passing for only `GH` scenario we can move this inside the module where it is needed.

Reply:
i imported at the extension level since we did the same for the az extension to allow it to be passed into multiple steps if needed. I could bring it into the OIDC generation command, but i think it is likely that other flows could use the git extension as well